esp/esp/program/modules/handlers/studentclassregmodule.py

```diff
@@ -453,12 +453,25 @@ def catalog_render(self, request, tl, one, two, module, extra, prog, timeslot=No
 
         categories = {}
         for cls in classes:
-            categories[cls.category_id] = {'id':cls.category_id, 'category':cls.category_txt if hasattr(cls, 'category_txt') else cls.category.category}
+            categories[cls.category_id] = {'id':cls.category_id, 'category':cls.category_txt if hasattr(cls, 'category_txt') else cls.category.category, 'symbol':cls.category.symbol}
+
+        # Is the catalog sorted by category? If so, by which aspect of category?
+        # Default is to sort by category symbol
+        cat_sort = 'category__symbol'
+        program_sort_fields = Tag.getProgramTag('catalog_sort_fields', prog)
+        if program_sort_fields:
+            cat_sort = program_sort_fields.split(',')[0]
+
+        cat_sort_split = cat_sort.split('__')
+        if cat_sort_split[0] == 'category' and cat_sort_split[1] in ['id', 'category', 'symbol']:
+            categories_sort = sorted(categories.values(), key = lambda cat: cat[cat_sort_split[1]])
```

Comment:
Could we rename "cat_sort" to be "catalog_sort"? "Cat" could be an abbreviation for categories and the distinction between cat_sort and categories_sort is a lot more clear if the first is catalog_sort.